Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM dwolla/jenkins-agent-core:alpine
+
+ENV SONAR_RUNNER_HOME=/usr/lib/sonar-scanner
+ENV SONAR_SCANNER_VERSION=4.4.0.2170
+
+USER root
+WORKDIR /usr/src
+
+RUN apk add --no-cache curl grep sed unzip nodejs nodejs-npm bash && \
+    npm install -g typescript && \
```

Comment:
Why does node and typescript need to be installed?  This looks like a jvm binary

Reply:
https://docs.sonarqube.org/latest/analysis/languages/javascript/